dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_item.tsx

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useState } from 'react';
+import { EuiTitle, EuiComboBox, EuiSpacer, EuiButton, EuiFieldText, EuiFlexItem, EuiFormRow, EuiIcon, EuiPanel, EuiText } from '@elastic/eui';
+import { useDispatch, useSelector } from 'react-redux';
+import { render as renderExplorerVis } from '../../../../../../event_analytics/redux/slices/visualization_slice';
+import { selectExplorerVisualization } from '../../../../../../event_analytics/redux/slices/visualization_slice';
+import { AGGREGATION_OPTIONS } from '../../../../../../../../common/constants/explorer';
+import { ButtonGroupItem } from './config_button_group';
+import { visChartTypes } from '../../../../../../../../common/constants/shared';
+
+export const DataConfigPanelItem = ({
+  fieldOptionList,
+  visualizations,
+  tabID
```

Comment:
same as above .useContext can be used.

Reply:
removed from props as we are using it from useContext